### PR TITLE
Add missing path in the Docker app.ini template

### PR DESCRIPTION
**Pull request by twang2218**
Some paths are missing in the Docker image's `app.ini` template, it will cause the `gitea` create files out of the `/data` directory. This PR added the missing paths.

Signed-off-by: Tao Wang <twang2218@gmail.com>

### CODE REVIEW

File: docker/etc/templates/app.ini [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
 
+[repository.local]
+LOCAL_COPY_PATH = /data/gitea/tmp/local-repo
+
 [repository.upload]
 TEMP_PATH = /data/gitea/uploads
 
 [server]
 APP_DATA_PATH = /data/gitea
+LFS_CONTENT_PATH = /data/gitea/data/lfs
```
Comment (lafriks) [inline docker/etc/templates/app.ini]: I don't think it is good default for LFS, I think better would be `/data/git/lfs` as repositories is stored in `/data/git/repositories` directory

Comment (twang2218) [reply docker/etc/templates/app.ini]: Yes, you are right. They are all `git` related stuff, so it's better they stay together. I'll change it to `/data/git/lfs` as you suggested.

**Comment by twang2218** [comment]
Updated with `LFS_CONTENT_PATH = /data/git/lfs`

**Comment by twang2218** [comment]
There is one more thing I want to mention, as all missing paths are added, the `/app/gitea/indexers` will not be created, and `gitea` binary is the only file in the `/app/gitea/` directory. Should we move the `gitea` binary to more standard place, such as `/usr/local/bin/`?

One benefit would be that, when user enter the container via `docker exec -it xxxx bash`, they can directly call `gitea` command, without full path `/app/gitea/gitea`.

**Comment by twang2218** [comment]
Added another commit of moving `gitea` binary from `/app/gitea/` to `/usr/local/bin/`

**Comment by silverwind** [comment]
> moving gitea binary from /app/gitea/ to /usr/local/bin/

maybe it's better to just symlink it?

**Comment by lunny** [comment]
@twang2218 why move gitea binary from `/app/gitea/` to `/usr/local/bin/`

**Comment by twang2218** [comment]
> why move gitea binary from /app/gitea/ to /usr/local/bin/

@lunny because 

1. `gitea` binary is the only file in the `/app/gitea/` directory after this changes, there is not much reason to keep this directory anymore.
2. moving the `gitea` binary to more standard place, such as `/usr/local/bin/`, will make `gitea` command globally available, especially useful when `docker exec -it gitea bash` to enter the container and run `gitea` commands without using the full path, `/app/gitea/gitea`.

If there are any other parts using hardcoded `/app/gitea/gitea` path, I can make a symlink there to has better backward compatibility.

**Comment by twang2218** [comment]
Rebased to resolve the conflict, and add symlink to `/app/gitea/gitea` for compatibility.

```dockerfile
# For backward compatibility
RUN mkdir -p /app/gitea/ && ln -s /usr/local/bin/gitea /app/gitea/gitea
```

_Updated: removed unnecessary `chmod`._

**Comment by daviian** [comment]
@twang2218 PR needs rebase to resolve conflicts.

**Comment by twang2218** [comment]
Rebased now.

**Comment by sapk** [comment]
~~~I will have to re-test this but doesn't actual docker image will put lfs content under `/data/gitea/lfs` ? Changing this default path will broke old install. This is not blocking but need to take in consideration and advertise.~~~ 
This will not break older version since the app.ini will not be regenerated.

**Review by sapk — APPROVED** [review]
Maybe we could add the config to activate related function (lfs, indexer) as env config. This could be done in an other PR.

**Review by techknowlogick — APPROVED** [review]

**Comment by techknowlogick** [comment]
I've just restarted Drone CI so hopefully it passes.

**Review by appleboy — APPROVED** [review]

**Pull request merged**